.github/workflows/coverage.yml

```diff
@@ -24,18 +24,22 @@ jobs:
           CONAN_BUILD_OPTIONS: |
             -o celix:enable_testing=True
             -o celix:build_all=True
-            -o enable_code_coverage=True
+            -o celix:enable_code_coverage=True
+            -o celix:enable_testing_for_cxx14=True
+            -o celix:enable_testing_dependency_manager_for_cxx11=True
         run: |
           #force reequire libcurl 7.64.1, due to a sha256 verify issue in libcurl/7.87.0
-          conan install . celix/ci -pr:b default -pr:h default -if build ${CONAN_BUILD_OPTIONS} -b missing --require-override=libcurl/7.64.1 --require-override=openssl/1.1.1s
+          conan install . celix/ci -pr:b default -pr:h default -if build ${CONAN_BUILD_OPTIONS} -b missing -b cpputest --require-override=libcurl/7.64.1 --require-override=openssl/1.1.1s
       - name: Build
         run: |
           conan build . -bf build --configure
           conan build . -bf build --build
       - name: Test with coverage
         run: |
           cd build
+          source activate_run.sh
```

Comment:
This fix an issue introduced by #472, which was reproduced by https://github.com/apache/celix/actions/runs/4085503505/jobs/7043556947